### ISSUE TITLE
vcsa 6.5 logs

### ISSUE THREAD

**Issue by ChKoehler** [sexilog/sexilog#46]
I have successfully configured our new VCSA 6.5 to Forward syslog-messages to sexilog v0_99g

I can see appliance-logmessages in Kibana, but they have not been parsed correctly:
no entries for message_program and hostname,
unparsed text in 'message_body':
<13>1 2017-03-23T13:16:45.654835+01:00 vcsa vmon 2237 - -  Executing op API_HEALTH on service rbd...

Any Idea?

Christian.

**Comment by djulztls** [comment]
Yes, same issue, looks like the filter file is not up to date for vcsa 6.5, please help !

**Comment by definitytechnologies** [comment]
@ChKoehler 

Did you ever figure this out?
